posthog/rate_limit.py

```diff
@@ -180,10 +185,6 @@ def get_cache_key(self, request, view):
 
         return self.cache_format % {"scope": self.scope, "ident": ident}
 
-    def team_is_allowed_to_bypass_throttle(self, team_id: Optional[int]) -> bool:
-        allow_list = get_team_allow_list(round(time.time() / 60))
-        return team_id is not None and str(team_id) in allow_list
-
```

Comment:
This should still remain, I'm not sure that all of these are bypassed for the query API specifically

Reply:
I don't remove it, just make it a standalone function so it can be used both here and in concurrency limit. You can see it above in the file

Reply:
Oh you're right sorry, juggling too many things today 🤦‍♀️